### ISSUE TITLE
NullReferenceException: When "Name" optionally not supplied to [Column] Attribute

### ISSUE THREAD

**Issue by z0nekill** [cajuncoding/SqlBulkHelpers#20]
Hello! I've been experimenting with your library in .NET 8 and I really like it - I'll cut straight to the chase:

https://github.com/cajuncoding/SqlBulkHelpers/blob/634737e8b6e71ad06ca52bf463c87c2e0d17e0f9/NetStandard.SqlBulkHelpers/SqlBulkHelper/QueryProcessing/SqlBulkHelpersProcessingDefinition.cs#L205

Has a minor bug where it presumes that the Attribute name is required, it is not. When calling `ToString` on the Dictionary indexer throws the NullReferenceException

Example:

```csharp
[Table("AppHit")]
public partial class AppHit
{
    [Key]
    [Column("HitID")]
    public Guid HitId { get; set; }

    [Column("SessionID")]
    public Guid SessionId { get; set; }

    // Issue => Name Optionally set
    [Column(TypeName = "datetime")]
    public DateTime Timestamp { get; set; }
}
```

In cases such as these I believe it would be acceptable to fall back to `propInfo.Name`. 

Unfortunately, .NET developers using Database First projects cannot force the Name to be required, as the above file is generated automatically.

**Comment by cajuncoding** [comment]
Ah, ok. I don’t use that attribute much so didn’t realize Name was optional. Yes then the default should definitely fall back as if no `Column` attribute was specified…because no name was specified.

Thanks for the heads up, lets get that stability issue resolved 👍

**Comment by cajuncoding** [comment]
Now resolved in v2.4.5 available in Nuget.

**Comment by z0nekill** [comment]
> Now resolved in v2.4.5 available in Nuget.

Thanks for your help - the library is a beast, and is handling significant loads in our production environments very well.
